hypothesis-python/src/hypothesis/extra/cli.py

```diff
@@ -32,24 +32,21 @@
 import sys
 from difflib import get_close_matches
 
-from hypothesis.extra import ghostwriter
-
-try:
-    import click
-except ImportError:
-
-    def main():
-        """If `click` is not installed, tell the user to install it then exit."""
-        sys.stderr.write(
-            """
-The Hypothesis command-line interface requires the `click` package,
+MESSAGE = """
+The Hypothesis command-line interface requires the `{}` package,
 which you do not have installed.  Run:
 
     python -m pip install --upgrade hypothesis[cli]
 
 and try again.
 """
-        )
+
+try:
+    import click
+except ImportError:
+    def main():
```

Comment:
```suggestion
except ImportError:

    def main():
```

My bad, I accidentally deleted a blank line that the formatter insists on here :confused: